### ISSUE TITLE
ListView does not expose its children to the eachDescendant function

### ISSUE THREAD

**Issue by NathanaelA** [NativeScript/NativeScript#1476]
ListView stores its children in two different named structures (one iOS and one on Android) and neither of them are used when you are looking for descendants using the view.eachDescendant function.

The ListView should add the _eachChildView function to its prototype so that eachDescendant can work.

**Comment by vbresults** [comment]
+1.  StackLayouts don't either.

**Comment by NathanaelA** [comment]
@vbresults - I'm pretty sure StackLayouts children are visible to eachChildView/eachDescendant.   What is making you think they aren't visible.

**Comment by vbresults** [comment]
require("ui/core/view").eachDescendant(StackLayoutView, function() { /** never reach here.  get an error about an _eachDescendant function missing **/ });

I did discover that it has a set of functions specific to StackLayout to iterate over children, though.

**Comment by enchev** [comment]
I believe that this is caused be the fact that ListView is virtual. Hey @hshristov can you comment here?

**Comment by NathanaelA** [comment]
@enchev - No, that is not the reason.   The reason really is because the ListView doesn't expose the standard child iterator.   I actually work around this in my `nativescript-dom` plugin and am about to iterate all the existing listview children perfectly fine.     So this "issue" actually doesn't affect anyone using my plugin; my plugins **view.getElementById** (and the getElementsByClassName/getElementsByTagName) will find elements that the built in "view.getViewById" will totally miss.    So this has been solved in the community since before I posted this bug report -- but this reason for this bug report is I believe this fix should be imho done in core.
